[00:20:07] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[00:20:07] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[00:51:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[00:51:22] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:22:36] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:22:36] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:53:52] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:53:52] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[02:25:20] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[02:25:20] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[02:56:40] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[02:56:40] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:28:02] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:28:02] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:59:26] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:59:26] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[04:30:51] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[04:30:51] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[05:02:17] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[05:02:17] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[05:33:46] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[05:33:46] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:05:13] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:05:13] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:36:41] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:36:41] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:08:11] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:08:11] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:39:52] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:39:52] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:11:23] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:11:23] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:42:53] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:42:53] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:14:28] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:14:28] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:45:58] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:45:58] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:17:32] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:17:32] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:49:16] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:49:16] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[11:20:55] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:20:55] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[11:52:36] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:52:36] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:35:29] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot/dist/vkbot.js:72:35)
[15:35:29] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[19:17:11] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot/dist/vkbot.js:72:35)
[19:17:11] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
